lib/presenter/Chart.js

```diff
@@ -484,56 +486,40 @@ class Chart extends Events {
     * interaction functions
     */
 
-    _initZoom() {
-        var opts = {
-            d3Svg             : this.d3Svg,
-            drawingSurface    : this._drawingSurface,
-            children          : this._childElementMap,
-            xRange            : this.xRange,
-            yRange            : this.yRange,
-            xAxis             : this._xAxis,
-            yAxis             : this._yAxis,
-            margins           : this.margins,
-            width             : this.width,
-            height            : this.height,
-            showXGrid         : this.xGrid,
-            showYGrid         : this.yGrid,
-            dataSeriesType    : this._aggregateDataRange.getXAxisType()
-        };
```

Comment:
Passing this monster options object wasn't really necessary. The only things that get added to it now are `zoomFixtureX` and `zoomFixtureY`. We honestly can probably knock those out at some point too, but I didn't want to spend too much more time refactoring.